### PR TITLE
MTE-5304: Add Syndicated Content Features to Solve.

### PR DESCRIPTION
**Pull request by pobtastic**
https://jira.rax.io/browse/MTE-5304

Drupal Views has a bunch of formatter options but ALL are text, even the true and false (!)

This is why I check for `"True"` as a string, as this is how the endpoint will display it 😭 

```
... "field_attribution_name":"David Stinemetze","field_syndicated_content":"True"} ...
```

### CODE REVIEW

**Comment by WidgetsBurritos** [comment]
Preview URL: http://mte-5304-syndicated-content.solve.dev.website.rackspace.com/solve

Login Url: http://mte-5304-syndicated-content.solve.dev.website.rackspace.com/user

_Note: If your branch name contains underscores or periods, replace them with hyphens prior to attempting to access the links above._

**Review by runrog — APPROVED** [review]
👍

**Comment by WidgetsBurritos** [comment]
Preview URL: http://mte-5304-syndicated-content.solve.dev.website.rackspace.com/solve

Login Url: http://mte-5304-syndicated-content.solve.dev.website.rackspace.com/user

_Note: If your branch name contains underscores or periods, replace them with hyphens prior to attempting to access the links above._

File: src/pages/solve-overview/template.html [hunk 1]
```diff
@@ -25,7 +25,8 @@ <h3 v-html="topic.desc"></h3>
             <h1 class="rsTl-feature-title"><a class="rsTl-feature-titleLink track-cta" :href="featured.first.alias">{{ featured.first.header }}</a></h1>
             <p class="rsTl-feature-p" v-html="$options.filters.truncate(featured.first.body, 200)"></p>
             <p class="rsTl-feature-p">
-              <span class="rsTl-feature-author">{{ featured.first.field_author }}</span> – {{ getDateString(featured.first.field_published_date) }}
+              <span class="rsTl-feature-author" v-if="featured.first.field_syndicated_content === 'True'">{{ featured.first.field_attribution_name }}</span>
+              <span class="rsTl-feature-author" v-else>{{ featured.first.field_author }}</span> – {{ getDateString(featured.first.field_published_date) }}
```
Comment (WidgetsBurritos) [inline src/pages/solve-overview/template.html:29]: As mentioned on the www PR. We need to add the company name here too per the ticket. 

> If the syndicated content checkbox is unchecked, display the company name next to the author's name
> - Use the Company field on Author, Leadership, and Event Speaker content types
> - For example Matt Stoyka / Rackspace

I believe we'll need to add company name to the api export to accomplish this.

**Review by WidgetsBurritos — CHANGES_REQUESTED** [review]

File: src/pages/solve-overview/template.html (hunk 1 above)
Comment (WidgetsBurritos) [reply src/pages/solve-overview/template.html:29]: This has been resolved on the Drupal side.  We're good here.

**Review by WidgetsBurritos — APPROVED** [review]
👍 LGTM

**Pull request merged**